### ISSUE TITLE
repository requires authentication

### ISSUE THREAD

**Issue by ankhazam** [foo4u/gradle-semver-plugin#1]
Provided repository is unavailable for anonymous access

**Comment by foo4u** [comment]
@ankhazam, sorry about that.  I may be deprecating this plugin or making significant changes to is b/c it's incompatible with Gradle's new [Maven Publishing Plugin](http://www.gradle.org/docs/current/userguide/publishing_maven.html).  Either way, I'll make it available on public repository shortly.

**Comment by ankhazam** [comment]
No problem, I cloned the codes and published them on my company internal Artifactory. I will look into the plugin you mensioned, thought I must admit that your semver plugin plays nicely with the gradle  artifactory-publish plugin.

**Comment by tkruse** [comment]
+1, same problem:

Received status code 401 from server: Unauthorized

**Comment by foo4u** [comment]
Sorry about that.  Recently made the repository private.  I will publish to Bintray or Maven Central shortly.  Stay tuned.

**Comment by ankhazam** [comment]
any updates on bintray/mavenCentral?

**Comment by foo4u** [comment]
Yes.  Just got approved for Maven Central distribution for net.rossillo group.  However, looking at [Gradle Plugins Repository](http://plugins.gradle.org) [submission guidelines](http://plugins.gradle.org/submit) it looks like it will have to be on JCenter.

Also there are two issues with this project as it currently stands:
1. It doesn't work with the newer maven-publish Gradle plugin
2. I think there's simpler solution (I will post this in the README today)
